test/e2e/test/elasticsearch/checks_es.go

```diff
@@ -141,9 +142,18 @@ func (e *esClusterChecks) CheckESNodesTopology(es estype.Elasticsearch) test.Ste
 			}
 			// match each actual node to an expected node
 			for nodeID, node := range nodes.Nodes {
-				nodeRoles := rolesToConfig(node.Roles)
+				// check if node present is coming from the expected stateful set based on its name
```

Comment:
maybe remove `present`?

Reply:
to be more explicit about why we do that:
`// ignore nodes coming from StatefulSets in the process of being downscaled`